app/build.gradle

```diff
@@ -21,13 +21,17 @@ android {
     }
 }
 
+androidExtensions {
+    experimental = true
```

Comment:
@KhinThiriSoe experimental is still here, can you remove that

Reply:
done. forgot to push